### DISCUSSION TITLE
Github Copilot in Visual Studio Code not generating code

### DISCUSSION THREAD

**Discussion by YongJun05** [community/community#134806] [GitHub Education]
### Hi everyone,

![Screenshot 2024-08-06 124718](https://github.com/user-attachments/assets/ed48319c-cebd-4723-98e5-dcf749765d77)
can anyone solve my problem? it's not working. It only shows genrating...

**Comment by Frid0l1n** [comment]
heyy there

Have you tried to uninstall and reinstall it again ? If this does not work please make sure you are running:

- the latest version of VScode
- you have "editor.inlineSuggest.enabled":true in your githubcopilot settings

If this doesnt work you could troubleshoot via

- the output from View > Command Pallete  > Github copilot : collect diagnostics

With the given output you can browse the web and troubleshoot further. If you want further help pls write a comment...

**Reply by YongJun05** [reply]
hi, I tried everything but it stills can't generating codes, but it is running , just can't show any code, please help, thank you

**Reply by Frid0l1n** [reply]
Is it possible to explain the problem or paste the error code here beneath thank you
